google-cloud-examples/src/main/java/com/google/cloud/examples/storage/objects/DownloadObject.java

```diff
@@ -35,12 +36,14 @@ public static void downloadObject(
     // String objectName = "your-object-name";
 
     // The path to which the file should be downloaded
-    // Path destFilePath = Paths.get("/local/path/to/file.txt");
+    // String destFilePath = "/local/path/to/file.txt";
+
+    Path path = Paths.get(destFilePath);
```

Comment:
What about inlining it?

```java
blob.downloadTo(Paths.get(destFilePath));
```

Or moving `Path path = Paths.get(destFilePath);` near `blob.downloadTo(path)`?